pyafq_cli.json

```diff
@@ -0,0 +1,501 @@
+        "name": "pruning thr",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[PRUNING_THR]",
+        "default-value": 6,
+        "optional": true
+    },
+    {
+        "id": "b0_threshold",
+        "name": "b0 threshold",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[B0_THRESHOLD]",
+        "default-value": 0,
+        "optional": true
+    },
+    {
+        "id": "prob_threshold",
+        "name": "prob threshold",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[PROB_THRESHOLD]",
+        "default-value": 0,
+        "optional": true
+    },
+    {
+        "id": "rng",
+        "name": "rng",
+        "type": "String",
+        "value-key": "[RNG]",
+        "default-value": "None",
+        "optional": true
+    },
+    {
+        "id": "segmentation_return_idx",
+        "name": "segmentation return idx",
+        "type": "String",
+        "value-key": "[SEGMENTATION_RETURN_IDX]",
+        "value-choices": ["false", "true"],
+        "default-value": "false",
+        "optional": true
+    },
+    {
+        "id": "filter_by_endpoints",
+        "name": "filter by endpoints",
+        "type": "String",
+        "value-key": "[FILTER_BY_ENDPOINTS]",
+        "value-choices": ["false", "true"],
+        "default-value": "true",
+        "optional": true
+    },
+    {
+        "id": "dist_to_aal",
+        "name": "dist to aal",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[DIST_TO_AAL]",
+        "default-value": 4,
+        "optional": true
+    },
+    {
+        "id": "n_points",
+        "name": "n points",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[N_POINTS]",
+        "minimum": 1,
+        "default-value": 100,
+        "optional": true
+    },
+    {
+        "id": "clean_rounds",
+        "name": "clean rounds",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[CLEAN_ROUNDS]",
+        "minimum": 1,
+        "default-value": 5,
+        "optional": true
+    },
+    {
+        "id": "distance_threshold",
+        "name": "distance threshold",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[DISTANCE_THRESHOLD]",
+        "minimum": 1,
+        "default-value": 5,
+        "optional": true
+    },
+    {
+        "id": "length_threshold",
+        "name": "length threshold",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[LENGTH_THRESHOLD]",
+        "minimum": 1,
+        "default-value": 4,
+        "optional": true
+    },
+    {
+        "id": "min_sl",
+        "name": "min sl",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[MIN_SL]",
+        "default-value": 20,
+        "optional": true
+    },
+    {
+        "id": "stat",
+        "name": "stat",
+        "type": "String",
+        "value-key": "[STAT]",
+        "value-choices": ["mean", "median"],
```

Comment:
> Do you mean stat? The statistic used in the cleaning algorithm? That's left a bit flexible here, but maybe we should restrict it to be one of 'median' or 'mean'. It really doesn't make much sense to use any other statistic to detect the central tendency of the coordinates of streamlines within a putative bundle.

@arokem I did indeed mean stat -- oops! ok, leaving as `mean` and `median` then. 👍

Reply:
Sounds good!